charts/tidb-operator/templates/scheduler-rbac.yaml

```diff
@@ -59,6 +58,15 @@ rules:
 - apiGroups: [""]
   resources: ["persistentvolumeclaims"]
   verbs: ["get", "list", "update"]
+# followng permissions are required if CSINodeInfo/AttachVolumeLimit features are enabled
```

Comment:
if CSINodeInfo/AttachVolumeLimit  are enabled (defaults true in 1.16.3), we need this permission

this is a temporary fix, I think it's better to bind our service account to `system:kube-scheduler` cluster role, then we don't need to care about it at all 

https://github.com/pingcap/tidb-operator/issues/1252